edge-modules/api-proxy-module/rust-sdk/azure-iot-mqtt/src/module.rs

```diff
@@ -277,7 +279,9 @@ impl futures_core::Stream for Client {
 						},
 
 						// Don't expect any subscription updates at this point
-						std::task::Poll::Ready(Some(Ok(mqtt3::Event::SubscriptionUpdates(_)))) => unreachable!(),
+                        std::task::Poll::Ready(Some(Ok(mqtt3::Event::SubscriptionUpdates(_)))) => unreachable!(),
```

Comment:
cargo fmt

Reply:
it completely changes the files and I wanted to keep the changes I made easy to be seen.